### ISSUE TITLE
Problems compiling with DDC

### ISSUE THREAD

**Issue by robbecker-wf** [ReactiveX/rxdart#118]
# Overview
rxdart does not seem to compile with the Dart Dev Compiler (DDC).  I'm using Dart 1.24.3.

dart2js works
```
> pub build test
```

ddc does not
```
> pub build test --web-compiler=dartdevc
```

I also tried:
```
pub serve --web-compiler=dartdevc example/web
```
and I get runtime errors when playing with the examples.

To try to find problems, I tried updating the analysis_options.yaml file to turn off implicit-casts:
```
analyzer:
  strong-mode:
    implicit-casts: false
    implicit-dynamic: true
```
And there appear to be a handful that I am not comfortable fixing.

**Comment by robbecker-wf** [comment]
Oops, Same problem as #117

**Comment by robbecker-wf** [comment]
see https://github.com/ReactiveX/rxdart/pull/119

**Comment by brianegan** [comment]
Fixed in 0.15.1
